deployment_scripts/deploy_windows.ps1

```diff
@@ -44,48 +44,40 @@ function Deploy-Windows([String] $monkey_home = (Get-Item -Path ".\").FullName,
     try
     {
         $version = cmd.exe /c '"python" --version  2>&1'
-        if ( $version -like 'Python 2.7.*' ) {
-            "Python 2.7.* was found, installing dependancies"
+        if ( $version -like 'Python 3.*' ) {
+            "Python 3.* was found, installing dependencies"
         } else {
             throw System.Management.Automation.CommandNotFoundException
         }
     }
     catch [System.Management.Automation.CommandNotFoundException]
     {
-        "Downloading python 2.7 ..."
+        "Downloading python 3 ..."
+        "Select 'add to PATH' when installing"
         $webClient.DownloadFile($PYTHON_URL, $TEMP_PYTHON_INSTALLER)
         Start-Process -Wait $TEMP_PYTHON_INSTALLER -ErrorAction Stop
-        $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine")
+        $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path","User")
         Remove-Item $TEMP_PYTHON_INSTALLER
         # Check if installed correctly
         $version = cmd.exe /c '"python" --version  2>&1'
         if ( $version -like '* is not recognized*' ) {
-            "Python is not found in PATH. Add it manually or reinstall python."
+            "Python is not found in PATH. Add it to PATH and relaunch the script."
             return
         }
     }
 
-    # Set python home dir
-    $PYTHON_PATH = Split-Path -Path (Get-Command python | Select-Object -ExpandProperty Source)
-
-    # Get vcforpython27 before installing requirements
```

Comment:
None of our requirements now require the VCRedist for execution?

Reply:
I don't really know what it was used for, but as far as I tested/googled we don't need it.